### DISCUSSION TITLE
Fundamentos do ASP.NET 6 - [HttpPut ("/{string}")]

### DISCUSSION THREAD

**Discussion by wlsouzaa** [balta-io/forum#511] [Tira Dúvidas] [answered]
Bom dia, estou tentando realizar um HttpPut mas não estou conseguindo, queria colocar no EndPoint ao inves do "Id" como de costume obter uma String "name" para realizar a pesquisa com o LIKE mas não estou conseguindo colocar o "{name: string}" na query do EF core da um erro que não consigo descobrir e nada na internet me da esta resposta.
![1](https://github.com/balta-io/forum/assets/137619531/f0cb1ba8-9aad-48fe-9b0d-37c03989d67a)
![2](https://github.com/balta-io/forum/assets/137619531/78a0559c-7bbd-456a-af4b-f8f762ba257f)
Se puder me explicar detalhadamente a solução eu agradeceria muito, pois ainda não memorizei os verbos do C#.
Desde ja grato.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @noobowisk, como vai?

Acredito que faltou um `.ToListAsync()` no fim do `Where`, não?

**Reply by wlsouzaa** [reply]
Muito Obrigaduuuuu, estou desde ontem a tarde pesquisando sobre isso, mas só acha sobre windows form...
Vou facilitar muito a vida do front, pois ele não vai ter q receber um lista monstruosa para tratar....
![3](https://github.com/balta-io/forum/assets/137619531/271d3118-8882-4165-aa6d-2aaeeeb061d8)
![4](https://github.com/balta-io/forum/assets/137619531/84ca3c7e-7c00-44dc-901a-232e946426a8)
